document-service-api/src/api/openapi.yaml

```diff
@@ -0,0 +1,205 @@
+---
+openapi: 3.0.0
+info:
+  title: Appeals Document Service API Documentation
+  version: 1.0.0
+  license:
+    name: www.foundry4.com
+servers:
+  - url: http://localhost:3001/
+paths:
+  "/api/v1/{applicationId}":
```

Comment:
Should the path be something like /api/v1/documents...?

Reply:
No. This is the document service, so that's the root level